recipes/piranha/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.5" %}
+
+package:
+  name: piranha
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
```

Comment:
Could change to `piranha-{{ version }}.tar.gz` to make better use of the source cache.